### PR TITLE
[ISSUE #3223]Optimize  startup script to support greater jdk version

### LINKED ISSUE #3223: support start namesrv/broker using higher version jdk
Currently we can not start broker due to Java 9 have deprecated -Djava.ext.dirs options.

### PR DESCRIPTION
**Pull request by ltamber**
**Make sure set the target branch to `develop`**

fix #3223 

## What is the purpose of the change

support start namesrv/broker using later jdk version (9,10,11,15,16)

## Brief changelog

1. use java classpath instead of -Djava.ext.dirs
2. use BouncyCastle because java 15 removed `sun.security.x509` package.
3. use different java options in different java version

## Verifying this change

tested in centos 7, using java version (8, 9, 10, 11, 15, 16).

### CODE REVIEW

**Review by caigy — APPROVED** [review]

File: acl/pom.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <parent>
         <groupId>org.apache.rocketmq</groupId>
         <artifactId>rocketmq-all</artifactId>
-        <version>4.9.1-SNAPSHOT</version>
+        <version>4.9.2-SNAPSHOT</version>
```
[the remark this answers is not in the document]
Comment (ltamber) [reply acl/pom.xml:16]: Sure, I'll fix it soon.

**Review by yuz10 — APPROVED** [review]

**Comment by CodePlayer** [comment]
@ltamber This PR has not been merged. Is it because some checks failed?

**Comment by ltamber** [comment]
> @ltamber This PR has not been merged. Is it because some checks failed?

The failed check is the unit test ratio decreased, I think the impact to the coverage will be small because of the main modification is the script.

**Comment by CodePlayer** [comment]
> The failed check is the unit test ratio decreased, I think the impact to the coverage will be small because of the main modification is the script.
@ltamber  I resolved this issue by merging your commits locally. Many thanks !
@yuz10  I hope this PR can be merged soon, because so many developers have encountered this problem.

**Comment by ltamber** [comment]
@duhenglucky PTAL

**Review by RongtongJin — APPROVED** [review]

**Review by duhenglucky — APPROVED** [review]

**Pull request merged**